### ISSUE TITLE
For some reason I can't enable Speedhack on Cheat Engine

### ISSUE THREAD

**Issue by LoreW001** [cheat-engine/cheat-engine#2722]
I download Cheat Engine because I wanted to use Speedhack but every time I click to enable it an error shows up. It says "error in injectCEHelperLib(): C Error :Incorrect tcc library". Anybody know how I can fix this error and enable Speedhack. I would appreciate it very much.

![Screenshot 2023-10-19 020429](https://github.com/cheat-engine/cheat-engine/assets/148425612/99c7b397-fa19-4800-9445-62e6a48c2695)

**Comment by bb33bb** [comment]
as far as i know, tcc lib need to compile

**Comment by LoreW001** [comment]
Oh I figured it out. In case anyone else has the same problem as me, I open tcc.sln with Visual Studio, Under the Project menu tab, retarget the solution, Under the Build menu tab, open the Configuration manager, Change your active solution configuration to "Output to 32 (Release)," and your active solution platform to Win32, Under the Build menu tab, build the solution and repeat steps 4 and 5, but with the other 3 combinations of configurations 32 (Release) and 64 (Release), and platforms Win32 and x64. Found this info from an old issue. Thanks for trying to help, bb33bb.
